sources/Interop/Windows/um/errhandlingapi/Windows.cs

```diff
@@ -0,0 +1,77 @@
+// Copyright © Tanner Gooding and Contributors. Licensed under the MIT License (MIT). See License.md in the repository root for more information.
+
+// Ported from um/errhandlingapi.h in the Windows SDK for Windows 10.0.19041.0
+// Original source is Copyright © Microsoft. All rights reserved.
+
+using System.Runtime.InteropServices;
+
+namespace TerraFX.Interop
+{
+    public static unsafe partial class Windows
+    {
+        [DllImport("kernel32", ExactSpelling = true)]
+        public static extern void RaiseException([NativeTypeName("DWORD")] uint dwExceptionCode, [NativeTypeName("DWORD")] uint dwExceptionFlags, [NativeTypeName("DWORD")] uint nNumberOfArguments, [NativeTypeName("const ULONG_PTR *")] nuint* lpArguments);
+
+        [DllImport("kernel32", ExactSpelling = true)]
+        [return: NativeTypeName("LONG")]
+        public static extern int UnhandledExceptionFilter([NativeTypeName("struct _EXCEPTION_POINTERS *")] EXCEPTION_POINTERS* ExceptionInfo);
+
+        [DllImport("kernel32", ExactSpelling = true)]
+        [return: NativeTypeName("LPTOP_LEVEL_EXCEPTION_FILTER")]
+        public static extern delegate* unmanaged<EXCEPTION_POINTERS*, int> SetUnhandledExceptionFilter([NativeTypeName("LPTOP_LEVEL_EXCEPTION_FILTER")] delegate* unmanaged<EXCEPTION_POINTERS*, int> lpTopLevelExceptionFilter);
+
+        [DllImport("kernel32", ExactSpelling = true)]
+        [return: NativeTypeName("DWORD")]
+        public static extern uint GetLastError();
+
+        [DllImport("kernel32", ExactSpelling = true)]
+        public static extern void SetLastError([NativeTypeName("DWORD")] uint dwErrCode);
```

Comment:
These APIs are problematic to expose and basically should never be called from .NET

The JIT can call many different native APIs in the background, even from say a GC interrupt, and cause `LastError` to become modified.

Reply:
Noted, willfix